scripts/testpools.bash

```diff
@@ -109,7 +109,8 @@ POOLS="$POOLS stratum+tcp://ETH_WALLET.WORKERNAME@kr.sparkpool.com:3333"
 POOLS="$POOLS stratum+tcp://ETH_WALLET.WORKERNAME@na-east.sparkpool.com:3333"
 POOLS="$POOLS stratum+tcp://ETH_WALLET.WORKERNAME@na-west.sparkpool.com:3333"
 POOLS="$POOLS stratum+tcp://ETH_WALLET.WORKERNAME@tw.sparkpool.com:3333"
-
+# whalesburg
+POOLS="$POOLS stratum1+tcp://ETH_WALLET.WORKERNAME@proxy.pool.whalesburg.com:8082"
```

Comment:
Please use in the script  **stratum** and not stratum**1** in the test script!
As ethminer now supports autodetection of stratum protocol we possible remove `startum1` and `stratum2` prefix.
As a lot of users does not switch to the README.md based on their used version we've not updated the POOL_EXAMPLES_ETH.md. So currently POOL_EXAMPLES_ETH.md matches also version ethminer-0.15.
But to be prepared for the future it would be good to use the new syntax in the test script.
(Maybe you also try `git rebase -i master` to join your commits into one)